pages/trust-admin.js

```diff
@@ -9,23 +9,23 @@ import NumberTile from "../src/components/NumberTile";
 import Text from "../src/components/Text";
 import AnchorLink from "../src/components/AnchorLink";
 import { TRUST_ADMIN } from "../src/helpers/userTypes";
+import moment from "moment";
+import momentDurationFormatSetup from "moment-duration-format";
+momentDurationFormatSetup(moment);
```

Comment:
Non-blocking: We could wrap this in a helper like we've done with formatting date and time.

Reply:
Ye I agree it would be good to get the duration calculation out of the page. Since it's only being used in the usecase, do we think that would be a suitable place to put it for now ? (also because it's formatted specifically for this usecase so that the string fits within the number tile)

Reply:
Works for me! We can always move it out if it's needed elsewhere.